Tests/Source/Model/User/ZMUserTests.swift

```diff
@@ -718,3 +718,55 @@ extension ZMUserTests {
     }
 }
 
+// MARK: - Verifying user
+extension ZMUserTests {
+    
+    func testThatUserIsVerified() {
```

Comment:
Let's be more specific when naming the tests

```suggestion
    func testThatUserIsVerified_WhenBothSelfUserAndOtherUserIsTrusted() {
```